ibm/service/kubernetes/data_source_ibm_container_vpc_cluster.go

```diff
@@ -39,6 +40,12 @@ func DataSourceIBMContainerVPCCluster() *schema.Resource {
 					"ibm_container_vpc_cluster",
 					"name"),
 			},
+			"wait_till": {
```

Comment:
I think we will have one challenge here adding wait_till in data source where user can't control the timeout how long they can wait.
 What is the default timeout it is taking as part of datasource